libs/website/feature/events/contact/contact.stories.tsx

```
@@ -0,0 +1,144 @@
+Success.args = {
+  isSuccess: true,
+}
+Success.parameters = {
+  docs: {
+    description: {
+      story: 'Contact form after successful submission',
+    },
+  },
+}
+
+export const Error = Template.bind({})
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)